utils/passkeys/create-suborganization.ts

```diff
@@ -0,0 +1,9 @@
+import { createPasskey } from "@turnkey/react-native-passkey-stamper";
+import { createTurnkeySubOrganization } from "../api";
+
+export async function createSubOrganization(
+  authenticatorParams: Awaited<ReturnType<typeof createPasskey>>
+) {
+  const res = await createTurnkeySubOrganization(authenticatorParams);
```

Comment:
i think since sub org is already a turnkey specific implementation detail, we don't need this wrapper